### ISSUE TITLE
When an SSH key is added during instance create, automatically check the box for it

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#2347]
I added a key in the form, which is nice, but it was unchecked by default, which is silly because you pretty much always want to use it right then.

**Comment by david-crespo** [comment]
I have a vague sense we may have done this and forgotten to close the issue.

**Comment by fakemonster** [comment]
Yup, handled in https://github.com/oxidecomputer/console/pull/3173
